lib/game.rb

```diff
@@ -22,9 +23,21 @@ def start
 
   def play
     # use a while loop later to keep game going
-    print_board
-    @board.place_mark(@player_1.mark, @player_1.get_input(@display))
-    print_board
+    while @game_over == false
+      print_board
+      @board.place_mark(@active_player.mark, @active_player.get_input(@display))
+      print_board
+      switch_player
+      @board.place_mark(@active_player.mark, @active_player.get_input(@display))
+      print_board
+      @game_over = true
+    end
+  end
+
```

Comment:
I don't think you need a while loop now or the game_over flag.  All you need at this stage is print_board, place_mark, print_board, place_mark, print_board